pkg/adapter/runtime.go

```diff
@@ -341,3 +341,8 @@ func IsImageNotFound(err error) bool {
 	}
 	return false
 }
+
+// HealthCheck is a wrapper to same named function in libpod
+func (r *LocalRuntime) HealthCheck(c *cliconfig.HealthCheckValues) (libpod.HealthCheckStatus, error) {
```

Comment:
Why are we passing in a HealthCheckValues here? we're only using a single argument (the container ID)?

Reply:
Should just pass in `id string` or similar

Reply:
because this is likely to expand ... like for --all, -l, etc

Reply:
Can't use args[0] anyways - if they pass in nothing everything explodes. You'd be better off parsing it in cmd/podman so we can share the code to make sure there are a sensible number of arguments, flags are properly parsed, etc

Reply:
you should not that parsing in cmd/podman/healthcheck already verifies one and only arg is being passed. its in the cmd definition.